### PR TITLE
LOK-2336: Clean up Tilt file and combine minion and minion-dev

### PR DESCRIPTION
**Pull request by Umairk95**
## Description
We created lokahi-minion and lokahi-minion-dev charts since Tilt wasn’t creating the required certificates properly lokahi-minion-dev .
lokahi-minion-dev is only used by Tilt.
Now Tilt is using lokahi-minion chart and certificate is working.

## Jira link(s)
- https://opennms.atlassian.net/browse/LOK-2336

## Flagged for review
Kindly run and test  Tilt  from uk/jira/LOK-2336 branch before approving PR .  
Tilt is working fine and certificates are working ?

## Checklist
* [ ] Follows Lōkahi's [development guidelines.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts.
* [ ] Notify documentation team of any changes to names of screens or features (affects URLs).

### CODE REVIEW

File: tmp-minion/minion.yaml
Comment (mershad-manesh) [inline tmp-minion/minion.yaml:1]: I suspect we don't need this file?

Comment (Umairk95) [reply tmp-minion/minion.yaml:1]: Yes,
removed this file

File: Tiltfile [hunk 1]
```diff
@@ -286,7 +286,14 @@ ssl_check('onmshs.local', 1443, tries=300,
...
         'ui:prod',
-        'minion-gateway',
+        'cert-manager',
+        'citus',
+        'citus-worker',
+        'ingress-nginx-repo',
+        'minion-certificate-manager',
+        'keycloak',
+        'inventory',
+        'datachoices'
```
Comment (cgorantla) [inline Tiltfile:296]: Does ssl_check need all of these dependencies ?

File: Tiltfile [hunk 2]
```diff
@@ -425,11 +432,20 @@ k8s_yaml(
...
-        values=['./tilt-helm-values.yaml'] + cfg.get('values', []),
+        'charts/lokahi-minion',
+        values=['./charts/lokahi-minion/values.yaml'] + cfg.get('values', []),
     )
+    
 )
 # Builds #
+## Minion Location ##
+local_resource(
+    'minion-location',
```
[the remark this answers is not in the document]
Comment (Umairk95) [reply Tiltfile:443]: yes minion-default is better, changed to minion-default

File: Tiltfile [hunk 3]
```diff
@@ -575,11 +591,12 @@ jib_project_multi_module(
     'minion-gateway',
     'opennms-minion-gateway',
     port_forwards=['16080:8080', '16050:5005'],
-    resource_deps=['shared-lib', 'citus-worker'],
+    resource_deps=['minion-location'],
```
[the remark this answers is not in the document]
Comment (Umairk95) [reply Tiltfile:594]: yes,  otherwise it gives warning and  tilt ci fails.

File: Tiltfile [hunk 4]
```diff
@@ -606,24 +623,34 @@ k8s_resource(
     port_forwards=['12022:8101', '12080:8181', '12050:5005'],
     labels=['minion'],
     trigger_mode=TRIGGER_MODE_MANUAL,
-    resource_deps=['shared-lib'],
+    resource_deps=['minion-location'],
```
[the remark this answers is not in the document]
Comment (Umairk95) [reply Tiltfile:626]: No, at this point minion doesn't need shared-lib

File: Tiltfile [hunk 5]
```diff
@@ -606,24 +623,34 @@ k8s_resource(
...
+#     text='Rollout restart minion',
+#     resource='minion',
+#     argv=[
+#         'kubectl',
+#         '--context', k8s_context(),
+#         '-n', k8s_namespace(),
+#         'rollout', 'restart', 'deployment', 'opennms-minion',
+#     ],
+#     icon_name='sync',
+# )
```
[the remark this answers is not in the document]
Comment (Umairk95) [reply Tiltfile:644]: commented code removed.

**Review by smunir-onms — COMMENTED** [review]
LGTM

**Pull request closed without merging**